Assets/ZSSRichTextEditor.js

```diff
@@ -48,6 +51,16 @@ ZSSEditor.defaultParagraphSeparator = 'p';
  */
 ZSSEditor.init = function() {
     
+    // Change a few CSS values if the device is an iPad
+    ZSSEditor.isiPad = (navigator.userAgent.match(/iPad/i) != null);
+    if (ZSSEditor.isiPad) {
+        $(document.body).css('padding-left', '80px');
+        $(document.body).css('padding-right', '80px');
+        $('#zss_field_title').css('font-size', '1.3em');
+        $('#zss_field_content').css('font-size', '1.125em');
+        $('#zss_field_content').css('line-height', '28px');
```

Comment:
Would it be make sense to move these styles into the css file, and then update `document.body.className` with the relevant class for a true IF check?